bridge/shared/models/webhook-config.ts

```diff
@@ -10,8 +24,9 @@ export class WebhookConfig implements wc {
   public method: WebhookConfigMethod;
   public url: string;
   public payload: string;
-  public header: { name: string; value: string }[];
+  public header: WebhookHeader[];
   public proxy?: string;
+  public secrets?: WebhookSecret[];
```

Comment:
Currently, the secrets are also sent to the client but the client does not need them